SpectrometerHDF5OutputFile.cpp

```diff
@@ -102,14 +107,11 @@ cSpectrometerHDF5OutputFile::cSpectrometerHDF5OutputFile(const std::string &strF
     H5Pclose(datasetPropertiesVis);
     H5Pclose(datasetPropertiesStokes);
 
+    // Read noise diode data from csv files
+    const string stNdFilesPath = "../../NoiseDiode/";
+    addNoiseDiodeData(stNdFilesPath);
+
     //Read antenna config information from ini file
-    pt::ptree ObsInfo;
-    pt::ini_parser::read_ini("ObservatoryInfo.ini", ObsInfo);
-    setAntennaName(ObsInfo.get<string>("Antenna.name"));
-    setAntennaLatitude(ObsInfo.get<string>("Antenna.latitude"));
-    setAntennaLongitude(ObsInfo.get<string>("Antenna.longitude"));
-    setAntennaAltitude(ObsInfo.get<string>("Antenna.altitude"));
-    setAntennaDiameter(ObsInfo.get<string>("Antenna.diameter"));
     setAntennaBeamwidth(ObsInfo.get<string>("Antenna.beamwidth"));
```

Comment:
It looks like you removed ObsInfo and parsing of the ini file but you are still using it here.